src/main/java/com/es/phoneshop/model/dao/ArrayListProductDao.java

```diff
@@ -1,4 +1,7 @@
-package com.es.phoneshop.model.product;
+package com.es.phoneshop.model.dao;
```

Comment:
It's better to put ArrayListProductDao class in sub-package of **dao** package and call it **impl**

Reply:
Please check this comment again.